src/Core/src/Platform/Windows/TransformationExtensions.cs

```diff
@@ -8,15 +8,15 @@ public static class TransformationExtensions
 	{
 		public static void UpdateTransformation(this FrameworkElement frameworkElement, IView view)
 		{
-			double anchorX = view.AnchorX;
-			double anchorY = view.AnchorY;
-			double rotationX = view.RotationX;
-			double rotationY = view.RotationY;
-			double rotation = view.Rotation;
-			double translationX = view.TranslationX;
-			double translationY = view.TranslationY;
-			double scaleX = view.Scale * view.ScaleX;
-			double scaleY = view.Scale * view.ScaleY;
+			double anchorX = double.IsFinite(view.AnchorX) ? view.AnchorX : default;
```

Comment:
Seems like a simple fix, right? Well, the default values for AnchorX and AnchorY is 0.5 (not default/0) and for ScaleX and ScaleY it is 1 (again, not default/0)

Reply:
Right, updated.